R/package.r

```diff
@@ -209,3 +214,16 @@ package_vignettes <- function(path = ".") {
     description = desc
   )
 }
+
+find_template_config <- function(package) {
+  if (is.null(package)) {
+    return(NULL)
+  }
+
+  config <- path_package_pkgdown(package, bs_version = NULL, "_pkgdown.yml")
```

Comment:
I went back and forth reg `bs_version` here. I assume that the template configuration file could set the `bs_version`, which is why there can only be one template configuration file per template package.

Reply:
Yeah, I think that makes sense